HISTORY.rst

```diff
@@ -2,6 +2,21 @@
 History
 =======
 
+2.5.8 (2019-09-13)
+------------------
+Breaking changes:
+
+* Patch selenium to convert `executeScript` to `executeAsyncScript`. This is a workaround for the `executeScript` issue in chromedriver 77.
+
+New features:
+
+* `LoadData` now supports the key `action: update` to perform a Bulk API update job
+* `LoadData` now supports an `after: <step name>` on a lookup entry to defer updating that lookup until a dependent sObject step is completed.
+* `GenerateMapping` now handles self-lookups and reference cycles by generating `after:` markers wherever needed. 
+
+Issues closed:
```

Comment:
Did you preview the restructuredText rendering? I think a blank line is required before the list starts.

Reply:
Didn't catch the newline, and yes, double backticks needed (I'm so used to single ones from PR comments and quip.